docs/UserGuide.md

```diff
@@ -373,6 +373,30 @@ Examples:
 * `eremovev vid/1 eid/1` 
   * removes the volunteer with id 1 from the event with id 1.
 
+### Undo a Command: `undo`
+
+Undoes an undo-able command
+
+* Undo-able commands include:
```

Comment:
hmm would be good if can standardize the format with our other features tho can put this under restrictions

Reply:
ya